readme.md

```diff
@@ -82,6 +82,19 @@ To deploy app on embedded server type:
 In the browser the app will open by default on:
 `localhost:8080`
 
+To check if the application works correctly:
+
+* enter the address in the browser
+
+[subs = + quotes]
+----
+https: // ip_address: port / health
+----
+
+Check the status of the application.
+The correct one is `UP`
+
+
```

Comment:
![image](https://user-images.githubusercontent.com/45402204/147442638-b5307859-a70f-4906-921f-68805efe948b.png)
I don't think it should look like this 😄